### DISCUSSION TITLE
Theme Presets

### DISCUSSION THREAD

**Discussion by a-y-u-s-h** [chakra-ui/chakra-ui#1971] [General]
# Feature request

## Theme Presets

I recently discovered Chakra UI and I love it functionality-wise. I was looking for something similar to [Evergreen UI ](https://evergreen.segment.com/components) in API and Chakra UI is very similar, except it supports more elements and better ones in v1.0.0, which ships with TailwindCSS preset by default. It would be wonderful if Chakra community provides a package called maybe @chakra-ui/presets like @theme-ui/presets which stores multiple themes (contributed by community or imitated from other UI libraries, such as material). 

## Describe the solution you'd like

Theme presets are basically multiple theme files packaged up, and maybe exported as a wrapped up React context. Having a separate package for community maintained theme files would give a lot of power for developers to pick between themes that others have worked on. I was specifically looking for Evergreen UI theme file for Chakra UI, so that I can make my components look and feel exactly like Evergreen's but they now have extra functionality and flexibility that Chakra UI provides (Evergreen is does not have a good themeing support and some components are absent like a slider, number input, etc - never implemented and are seen as low priority since last few years).

## Describe alternatives you've considered

Theme UI, but it's too much work. Chakra UI v1 is the best I've seen in terms of flexibility. Theme UI has a [preset package](https://theme-ui.com/packages/presets).

**Comment by chasinhues** [comment]
Sorry for the lack of responses on our part @a-y-u-s-h, and thanks for the suggestion! Love the idea of theme presets in Chakra! 😍 

I've also noticed a lot of discussion in the community in the past couple of months especially around people searching for or creating their own public Chakra-based component libraries.
